wazo_webhookd/plugins/subscription/celery_tasks.py

```diff
@@ -32,15 +30,15 @@ def hook_runner_task(task, hook_uuid, ep_name, config, subscription, event):
     logger.info("running hook %s (%s) for event: %s", ep_name, hook_uuid, event)
 
     try:
-        event_name = event['data']['name']
+        event_name = event['name']
```

Comment:
It should be possible to add a test to avoid this kind of typo?

Reply:
It's only used by logger, it will be a pain de test that.